src/data/security-quiz.ts

```diff
@@ -1,14 +1,36 @@
 const securityQuiz = [
   {
     Question:
-      "In digital security, what is a Man in the Middle Attack?",
-    Answer: "an attack where hackers can spy on your internet traffic and modify it",
-    Distractor1: "an attack where hackers can send emails with fake links",
-    Distractor2: "an attack where hackers can encrypt your files and demand payment",
-    Distractor3: "an attack where hackers can flood the system with network requests",
+      "Which of the following is a mechanism to ensure non-repudiation?",
+    Answer: "Assymetric-key encryption",
+    Distractor1: "MD5",
+    Distractor2: "Symmetric-key encryption",
+    Distractor3: "Caesar cipher",
     Explanation:
-      "Free wifi hotspots are often targeted by hackers and can put you at risk of MitM attacks (Man in the Middle Attacks) where a hacker spies on your internet traffic, and may even modify it without you knowing.",
-    Link: "https://www.freecodecamp.org/news/personal-digital-security-an-intro/",
+      "Assymetric-key encryption is a mechanism to ensure non-repudiation because it uses two keys,one public and one private.The public key is used to encrypt the message and the private key is used to decrypt the message.This ensures that the sender cannot deny sending the message because the message can only be decrypted with the private key.",
+    Link: "https://www.freecodecamp.org/news/the-cia-triad-confidentiality-integrity-and-availability-explained/",
```

Comment:
We do want to avoid completely replacing someone else's quiz question because then contributors won't want to participate anymore if they feel like their contributions don't matter and were removed from the quiz file.

It is fine to reword this Man in the Middle Attack and come up with different answers or distractors but we don't want to completely change it. And then you can add your other questions below it.

Reply:
I apologise; I didn't mean to replace the question.
It took place when the merge conflict problem was being resolved.
Could you kindly assist me in fixing this?

Reply:
I think the easiest fix would be to add their question right above yours.

You can throw this object right on top of your question, so it would be the first one on the list.

```
  {
    Question:
      "In digital security, what is a Man in the Middle Attack?",
    Answer: "an attack where hackers can spy on your internet traffic and modify it",
    Distractor1: "an attack where hackers can send emails with fake links",
    Distractor2: "an attack where hackers can encrypt your files and demand payment",
    Distractor3: "an attack where hackers can flood the system with network requests",
    Explanation:
      "Free wifi hotspots are often targeted by hackers and can put you at risk of MitM attacks (Man in the Middle Attacks) where a hacker spies on your internet traffic, and may even modify it without you knowing.",
    Link: "https://www.freecodecamp.org/news/personal-digital-security-an-intro/",
  },
```

Reply:
I think I added it below in my recent commit😅_